### PR TITLE
Fix questions display

### PR DESCRIPTION
**Pull request by skjnldsv**
- [ ] Questions gets duplicated in the db

### CODE REVIEW

**Comment by jotoeri** [comment]
@skjnldsv How far is your plan grown? Should i have a look into, too somehow?

**Comment by skjnldsv** [comment]
> @skjnldsv How far is your plan grown? Should i have a look into, too somehow?

You can if you want :)
I was waiting for @rullzer to do/finish th backend part, he was caught up in a customer call I think :thinking:

**Comment by rullzer** [comment]
Yes feel free to tackle this. I'm rather busy today

**Comment by jotoeri** [comment]
Ok... 
Would an Ansatz like you wrote the API->NewForm be reasonable to continue in NewQuestion & NewAnswer? (So one on each table)
Would result further in UpdateEvent, RemoveQuestion & Remove Answer, which alltogether replace the current writeForm?

**Comment by jotoeri** [comment]
Adding questions and answers works, but deleting does not. Does one of you see my problem?
In Create.vue->deleteQuestion the variables are ok, but between the axios.post-call and ApiController->deleteQuestion somehow the id gets lost and i just receive an id of 0?!

File: lib/Db/AnswerMapper.php [hunk 1]
```diff
@@ -95,4 +95,16 @@ public function deleteByForm(int $formId): void {
 
 	   $qb->execute();
 	}
+
+	public function findById(int $answerId): Answer {
```
Comment (jotoeri) [inline lib/Db/AnswerMapper.php]: Isn't this the same as just find($id)?

Comment (jotoeri) [reply lib/Db/AnswerMapper.php]: Sorry, my Error. I'm sometimes confused, which function is Base-Class and which one is Specialisation.

**Pull request merged**